backend/addcorpus/tests/test_reader.py

```diff
@@ -0,0 +1,30 @@
+import json
+import os
+from addcorpus.models import Corpus
+from addcorpus.reader import make_reader
+
+HERE = os.path.abspath(os.path.dirname(__file__))
+
+
+def test_make_reader_python(mock_corpus):
+    py_corpus = Corpus.objects.get(name=mock_corpus)
+    reader = make_reader(py_corpus)
```

Comment:
This is a bit confusing since `py_corpus` is not the Python definition.

```suggestion
    corpus = Corpus.objects.get(name=mock_corpus)
    reader = make_reader(corpus)
```